public/styles.css

```diff
@@ -0,0 +1,91 @@
+.eventTab{
+    overflow-x: hidden;
+    overflow-y: hidden ;
+}
+
+
+.eventImage{
+  height: 80%;
+  width: 100%;
+}
+
+.upcomingEventTitle{
+  margin: 40px;
```

Comment:
Try not to use px. It might be different on different screens. Use 'rem' instead

Reply:
> Try not to use px. It might be different on different screens. Use 'rem' instead

Okay